reana/reana_dev/utils.py

```diff
@@ -704,37 +724,56 @@ def translate_pep440_to_semver2(pep440_version):
         sys.exit(1)
 
 
-def bump_component_version(component, current_version, next_version=None):
-    """Bump to next component version."""
-    try:
-        version_files = get_component_version_files(component)
-        files_to_update = []
-
-        if version_files.get(HELM_VERSION_FILE):
-            next_version = next_version or bump_semver2_version(current_version)
-            files_to_update.append(version_files.get(HELM_VERSION_FILE))
-        elif version_files.get(PYTHON_VERSION_FILE):
-            next_version = next_version or bump_pep440_version(current_version)
-            files_to_update.append(version_files.get(PYTHON_VERSION_FILE))
-            if version_files.get(OPENAPI_VERSION_FILE):
-                files_to_update.append(version_files.get(OPENAPI_VERSION_FILE))
-        elif version_files.get(JAVASCRIPT_VERSION_FILE):
-            next_version = next_version or bump_semver2_version(current_version)
-            files_to_update.append(version_files.get(JAVASCRIPT_VERSION_FILE))
-
-        for file_ in files_to_update:
-            replace_string(
-                file_=file_,
-                find=current_version,
-                replace=next_version,
-                component=component,
+def bump_component_version(
+    component: str, next_version: str = None
+) -> (str, List[str]):
+    """Bump to next component version.
+
+    If next_version is set, it will be used for the component.
+    If not, version will be bump automatically from the current one.
+
+    :param component: component to bump
+    :param next_version: new PEP440 version
+    :type component: str
+    :type next_version: str
+
+    :return: next_version in SemVer2 format and list of modified version files
+    :rtype: str, List[str]
+    """
+    if next_version and not parse_pep440_version(next_version):
```

Comment:
Probably outside the scope of this PR, but I noticed that this check is actually not working:
```
cd reana
reana-dev git-create-release-commit -c . -v "0.8.0-alpha.3"
```
I've provided `SemVer2` version and it bumps `/reana/version.py` accordingly:
```
--- a/reana/version.py
+++ b/reana/version.py
@@ -13,4 +13,4 @@ This file is imported by ``reana.__init__`` and parsed by ``setup.py``.

 from __future__ import absolute_import, print_function

-__version__ = "0.8.0a2"
+__version__ = "0.8.0-alpha.3"
```

Reply:
Semver should be used only for Docker images and NPM packages and Helm charts... The PEP 0440 should be used for Python packages. We have a function that provides correspondence between them, so it should probably just be called in proper places...

Reply:
`-v` must be in PEP440 format, it is written in docstrings. Otherwise, Python next_version will not work (will need to write semver2 to pep440 converter or find a library to do it).

But I didn't expect that `parse_pep440_version` will work on `SemVer2`, should fail. Will check it.

Reply:
I found that the `packaging` package in Python can already do a translation from SemVer2 to PEP440 (`parse_pep440_version` is using the package). The question is how to decide when to use SemVer2 or PEP440 for git tags. E.g we have Helm and Python and we want to specify `next_version`. What format should be returned for git tags?